src/pages/home/report/ReportActionsList.js

```diff
@@ -126,10 +127,9 @@ class ReportActionsList extends React.Component {
         item: reportAction,
         index,
     }) {
-        // When the new indicator should not be displayed we explicitly set it to 0. The marker should never be shown above the
-        // created action (which will have sequenceNumber of 0) so we use 0 to indicate "hidden".
```

Comment:
NAB, but did we check to make sure that the new marker can't be displayed above the created action? The previous comment here suggests something about that - but I'm not too sure what we were worried about tbh.

Reply:
Yes, it will never show above the created action because if an action is a created action we render it via `ReportActionItemCreated` which doesn't even get passed the `shouldDisplayNewIndicator` prop whereas regular report actions are rendered via ReportActionItem. 

So actually looking at the code, we don't actually even need the null check. And because this component only ever gets passed visible actions, I don't think we need the `isDeletedAction` check either. I think we could actually just simplify this to: 

```const shouldDisplayNewIndicator = reportAction.reportActionID === this.props.newMarkerReportActionID;```

Reply:
Oh nice, that would be pretty sweet. Could we also set `newMarkerReportActionID` to an empty string instead of `null`?

Reply:
Yeah I don't see why not. Why is that better than null?

Reply:
Ok, updated!